README.md

```diff
@@ -25,6 +25,11 @@ Currently, all the plugins provided only work for remark versions **lesser than*
 
 This project uses [Jest][jest] for testing. It is recommended to use the locally installed version using `npx`, and run Jest in watch mode when developing `npx jest --watch --notify` (`--notify` sends desktop notifications when tests run).
 
+### Development
+
+1. `npm run build-demo`
+1. Open `packages/zmarkdown/public/index.html` in your browser
```

Comment:
The command `build-demo` was mostly made for internal use (ie. to release the demo on Github), I am not in favor of documenting it inside of the main README, but if you want you can update the `packages/zmarkdown` README, which contains the procedure to create development and production builds (at the end of the file).

Reply:
How do you test your code locally if not by running the demo? As a new contributor, I was stuck on that.

Starting the zmd server and making a POST request appears a bit laborious to me.

Reply:
ZdS, and all other users of ZMarkdown I am aware of, are using the server, therefore I rely exclusively on the server, and mostly use tests for checking problems. Implementing a new functionality is done using TDD most of the time; we have more than 1000 tests all over the project, so one is likely to fail is something broke. Occasionally, I use some tools [I have developed](https://github.com/StaloneLab/zmarkdown-experiments/tree/master/publisher) to compile a full tutorial, allowing to do some basic integration testing, but unit tests are usually enough.

To clarify what the client build is for: the live demo only, not development purposes, nor use in a production environment. We might have this kind of use someday, but it is not the current use (indeed, since the client build require some Mo to be loaded). Why do we keep it then? Because it has almost zero maintenance cost (did not break in more than five years, it is the first time), and some people use it occasionally to check Markdown syntax or see if a bug is indeed due to ZMd.

New developers are encouraged to use the server, for two quite simple reason: our tests will not run without a working server, and the server is what is used anyway. Actually, tests start the server when launched. For this PR, you had no choice but to deploy the demo (of course), however if you intend to work on something else, here is the recommended workflow:

1. write at least one test for your feature/bugfix;
2. write your code and build it with `npm run build`;
3. launch tests with `npm run test`;
4. repeat 2/3 until all tests (including yours) pass.

So regarding the idea of adding these directives to the main README, I still stand opposed, but I could accept something like "More documentation is located in the package/zmd README, especially parameters used for the server and usage of a client build" (not exactly this, since the sentence is weird).

Reply:
Thanks for the answer! It'd be nice to have a short tutorial in the README on how to add a simple example feature step by step.

Reply:
I guess you messed something up in your latest commits because Github only shows an addition of a link to the end of the main README which is not used inside of the content.

Reply:
Fixed